cmd/mesh-bridge/main.go

```diff
@@ -0,0 +1,112 @@
+// +build !js
+
+// mesh-bridge is a short program that bridges two Mesh nodes. This is useful in cases where
+// we introduce a network-level breaking change but still want the liquidity from one network
+//to flow to another
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/0xProject/0x-mesh/rpc"
+	"github.com/0xProject/0x-mesh/zeroex"
+	ethrpc "github.com/ethereum/go-ethereum/rpc"
+	"github.com/plaid/go-envvar/envvar"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	firstWSRPCAddressLabel  = "FirstWSRPCAddress"
+	secondWSRPCAddressLabel = "SecondWSRPCAddress"
+	maxReceiveBatch         = 10
+	receiveTimeout          = 10 * time.Second
```

Comment:
Optional Suggestion: Considering @albrow's original comment about wanting to make sure that orders are forwarded quickly after being received, I could see a case for making `receiveTimeout` an `envvar` so that it can be configured without changing the source code.